lib/modules/manager/pip-compile/extract.ts

```diff
@@ -23,7 +23,7 @@ import {
   sortPackageFiles,
 } from './utils';
 
-function matchManager(filename: string): SupportedManagers | 'unknown' {
+export function matchManager(filename: string): SupportedManagers | 'unknown' {
```

Comment:
should be moved to common in separate PR

Reply:
Opened #29359